students/Matt_Hudgins/Lesson02/series.py

```diff
@@ -0,0 +1,73 @@
+'''
+    File Name: series.py
+    Author: Matt Hudgins
+    Date created: 4/2/2018
+    Date last modified: 4/2/2018
+    Python Version 3.6.4
+'''
+
+
+def fibonacci(n):
+    """This function calculates the nth number of the Fibonacci
+    Series"""
+    if n == 1:
```

Comment:
This should produce the series starting with 0 index (like you did for Lucas below) so that `fib(0) == 0, fib(1) == 1` etc.